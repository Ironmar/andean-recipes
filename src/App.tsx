import React from "react";
import AsideImage from "./components/sections/AsideImage";
import MainContent from "./components/sections/MainContent";
import Navbar from "./components/sections/NavBar";
import RecipeProvider from "./context/RecipeContex";

const App: React.FC = () => {
  return (
    <RecipeProvider>
      <Navbar />
      <div className="container">
        <MainContent />
        <AsideImage />
      </div>
    </RecipeProvider>
  );
};
export default App;
