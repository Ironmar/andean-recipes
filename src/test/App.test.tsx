import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

import { configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import { createSerializer } from "enzyme-to-json";

import "@testing-library/jest-dom";
import App from "../App";
configure({ adapter: new Adapter() });
//@ts-ignore
expect.addSnapshotSerializer(createSerializer({ mode: "deep" }));

describe("Test in <App/>", () => {
  test("should be return the component correctly", () => {
    const wrapper = shallow(<App />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
