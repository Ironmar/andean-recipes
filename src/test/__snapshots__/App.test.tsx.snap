// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test in <App/> should be return the component correctly 1`] = `
<RecipeProvider>
  <Navbar />
  <div
    className="container"
  >
    <MainContent />
    <AsideImage />
  </div>
</RecipeProvider>
`;
